appstudio-utils/util-scripts/lib/fetch.sh

```diff
@@ -20,6 +20,8 @@ POLICIES_DIR=${POLICIES_DIR:-"$EC_WORK_DIR/policies"}
 POLICY_REPO=${POLICY_REPO:-"https://github.com/hacbs-contract/ec-policies.git"}
 POLICY_REPO_REF=${POLICY_REPO_REF:-"main"}
 
+# TODO: Fix this so it can handle the case when there is no cluster access. PR_NAME
+# is never used when we're processing attestations directly.
 # Pipeline run name
 PR_NAME=${1:-$( tkn pr describe --last -o name )}
 PR_NAME=$( echo "$PR_NAME" | sed 's|.*/||' )
```

Comment:
Just FYI, I moved this in #52 which isn't merged yet.

Reply:
Awesome! Just approved that PR.

Reply:
Merged. You can rebase when ready.